packages/wallets/src/features/cashier/modules/Transactions/Transactions.tsx

```diff
@@ -59,11 +62,19 @@ const Transactions = () => {
     }, [filterValue, isPendingActive]); // eslint-disable-line react-hooks/exhaustive-deps
 
     return (
-        <div className='wallets-transactions'>
-            <div className='wallets-transactions__header'>
+        <div
+            className={classNames('wallets-transactions', {
+                'wallets-transactions--crypto-mobile': wallet?.is_crypto && isMobile,
+            })}
+        >
+            <div
+                className={classNames('wallets-transactions__header', {
+                    'wallets-transactions__header--crypto-mobile': wallet?.is_crypto && isMobile,
```

Comment:
no need to repeat  modifiers - top level one should be enough to this header

Reply:
@wojciech-deriv you mean `className='wallets-transactions__header--crypto-mobile'`?

Reply:
Yea, I've done it this way:

<img width="465" alt="Screenshot 2023-11-07 at 15 18 20" src="https://github.com/binary-com/deriv-app/assets/119863957/36d5d752-fc96-48b6-b8d9-3ad4ff52773e">

Reply:
while not strictly defined by BEM standards, is good habit to keep modifiers only on top level "div" of component, as it limits the complexity of components and ppl tend to make them simpler

Reply:
Yes, I've removed this as well in this commit https://github.com/binary-com/deriv-app/pull/11184/commits/415eedfcfa9b206f4026d91601711588f90a9e09